tsconfig.spec.json

```diff
@@ -12,6 +12,9 @@
     "src/test.ts",
     "src/polyfills.ts"
   ],
+  "exclude": [
+    "src/app/tests/**.ts"
+  ],
```

Comment:
This change doesn't make alot of sense to me. I'm assuming the `tsconfig.spec.json` is the tsconfig that is used for tests, so `src/app/tests/**.ts` would be a part of that?

Reply:
Maybe i'm wrong, but my intention was to exclude this part of code from code coverage. But after checking this, you are right, this make no difference. So i will remove this part.